docs/standard/serialization/binaryformatter-migration-guide/migrate-to-protobuf-net.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "BinaryFormatter migration guide: Migrate to protobuf-net (binary)"
+description: "Migrate from BinaryFormatter to protobuf-net for binary serialization."
+ms.date: 5/31/2024
+no-loc: [BinaryFormatter, Serialization]
+helpviewer_keywords:
+  - "BinaryFormatter"
+  - "serializing objects"
+  - "serialization"
+  - "objects, serializing"
+---
+
+# Migrate to protobuf-net (binary)
+
+[protobuf-net](https://github.com/protobuf-net/protobuf-net) is a contract-based serializer for .NET code that writes data in the binary _protocol buffers_ serialization format engineered by Google.
+
+- By default, both public and non-public types are serializable. Every type needs to provide a parameterless constructor.
```

Comment:
It is more nuanced than that, and the clarifications are more than I can fit in a one-line suggestion; will push to the branch.

Reply:
done: f81998b839344d4863a64fd3a97e8a6883ef0f58

Reply:
@mgravell thank you!